pkg/metric/metric_timestamper.go

```diff
@@ -63,6 +63,10 @@ func NewMetricTimestamper(params MetricTimestamperParams) *MetricTimestamper {
 }
 
 func (m *MetricTimestamper) Process(batch *livekit.MetricsBatch) {
+	if m == nil {
+		return
+	}
```

Comment:
Using these to avoid `nil` checks at caller for cases where metrics is not enabled and hence these objects are not created.